MetasysServicesExampleApp/Program.cs

```diff
@@ -343,6 +349,67 @@ static void Main(string[] args)
                 Console.WriteLine($"\n Id: {item.Id}, Name: {item.Name}, ItemReference: {item.ItemReference}");
             }
 
+            Console.WriteLine("Enter object id to get alarm details: ");
+            string objectId = Console.ReadLine();
+            
+            Console.WriteLine(string.Format("\n Alarm details found for this object {0}", objectId));
+
+            string getAlarmsForObject;
+            AlarmFilter alarmFilterForObject = new AlarmFilter();
+
+            if (args.Length != 12)
+            {
+                Console.WriteLine("Please enter these parameters separated by space: Start Time, End Time, Priority range, Type, Exclude pending, Exclude acknowledged, Exclude discarded, Attribute, Category, Page, Page Size, Sort" +
+                    "\nRefer to the metasys-server/basic-services-dotnet README if you need help getting started.");
+                getAlarmsForObject = Console.ReadLine();
+                args = getAlarmsForObject.Split(' ');
+
+                alarmFilterForObject.StartTime = args[0];
+                alarmFilterForObject.EndTime = args[1];
+                alarmFilterForObject.PriorityRange = args[2];
+                alarmFilterForObject.Type = Convert.ToInt32(args[3]);
+                alarmFilterForObject.ExcludePending = Convert.ToBoolean(args[4]);
+                alarmFilterForObject.ExcludeAcknowledged = Convert.ToBoolean(args[5]);
+                alarmFilterForObject.ExcludeDiscarded = Convert.ToBoolean(args[6]);
+                alarmFilterForObject.Attribute = Convert.ToInt32(args[7]);
+                alarmFilterForObject.Category = Convert.ToInt32(args[8]);
+                alarmFilterForObject.Page = Convert.ToInt32(args[9]);
+                alarmFilterForObject.PageSize = Convert.ToInt32(args[10]);
+                alarmFilterForObject.Sort = args[12];
+            }
+			
```

Comment:
Create a private method and assign the value that is read from the console. It is repeating code so the same method will be called to get alarms, get alarm for network devices and objects.